api/src/main/java/org/openmrs/module/bedmanagement/BedManagementActivator.java

```diff
@@ -52,6 +52,7 @@ public void willStart() {
 	 */
 	public void started() {
 		log.info("bedmanagement Module started");
+		BedManagementProperties.initalize();
```

Comment:
@sanishmaharjan good idea to segregate the initialisation in an ad-hoc class.
👍